data/catalogs/deltares_data.yml

```diff
@@ -1,6 +1,8 @@
 ---
 meta:
-  root: p:/wflow_global/hydromt
+  roots:
+    - p:/wflow_global/hydromt
+    - /mnt/p/wflow_global/hydromt
   version: 2024.1.30
   hydromt_version: '>=0.9, <1.0'
```

Comment:
With these changes we should update the hydromt_version to >1.0 right? We should then also update the ref in the predefined catalogs.

Reply:
sort of. You're right that the refs need to be updated, but only once this lands on main right? this one is going to v1 not main, and when we merge v1 to main, we'll have to update the refs anyway. Am I correct?

Reply:
You're right, it's probably best to update them now. I've updated all the catalog versions, but I do wonder if we still need to update the `predefined_catalogs.yml` since this hasn't landed on main yet, and I'm not sure how it will handle things that aren't on main yet. What do you two think? if so I'll make an issue to update those refs when we finsih v1

Reply:
True, forgot that the latest version points to main and only the others to a commit number. Is there a reason why actually we point to main and not a commit number for the latest version?
But yes it's not so easy to update so let's open the issue to update for later.

Reply:
Why we point to main and not a specific commit I'm not sure. @DirkEilander knows most about that I think.